api/overview/azure/virtual-network.md

```diff
@@ -15,6 +15,8 @@ ms.service: multiple
 
 # Azure Virtual Network libraries for .NET
 
+## Overview
+The [Azure Virtual Network](https://docs.microsoft.com/en-us/azure/virtual-network/virtual-networks-overview) service enables you to securely connect Azure resources to each other with virtual networks (VNets). A VNet is a representation of your own network in the cloud. You can also connect VNets to each other, enabling resources connected to either VNet to communicate with each other. 

```

Comment:
Two options: either keep URL "as is" but remove the culture (/en-us/) portion, or just make it a partial URL starting at /azure ... i.e. /azure/virtual-network/virtual-networks-overview